model/pdata/common.go

```diff
@@ -513,7 +513,14 @@ func (am AttributeMap) Get(key string) (AttributeValue, bool) {
 
 // Delete deletes the entry associated with the key and returns true if the key
 // was present in the map, otherwise returns false.
+// Deprecated: Use Remove instead.
```

Comment:
Add the version to the deprecated statement

Reply:
done